ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Worker.java

```diff
@@ -600,6 +602,10 @@ public CompactorMR getMrCompactor() {
   }
 
   private void markFailed(CompactionInfo ci, String errorMessage) {
+    if (ci == null) {
+      LOG.warn("CompactionInfo client was null. Could not mark failed: {}", ci);
```

Comment:
why are you passing ci to the log if it's null? `Could not mark failed: {}", ci` - just remove this part of log

Reply:
Removed the parameter. Now it is: `LOG.warn("CompactionInfo client was null. Could not mark failed");`